addon/components/o-s-s/slider.ts

```diff
@@ -27,7 +28,8 @@ export default class SliderComponent extends Component<SliderComponentArgs> {
     min: this.args.min ?? 0,
     max: this.args.max ?? 100,
     step: this.args.step ?? 1,
-    autocomplete: this.args.autocomplete ?? 'on'
+    autocomplete:
+      this.args.autocomplete && AutocompleteValues.includes(this.args.autocomplete) ? this.args.autocomplete : 'on'
```

Comment:
Same as previous comment here

maybe you are getting a ts error ? maybe defaulting back to empty string using a nullish coalescing operator in the includes value ?